source/includes/mongodb-compatibility-table-php.rst

```diff
@@ -15,7 +15,20 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - ext 1.12 + lib 1.11
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     -
+     -
+     -
+     -
+
    * - ext 1.11 + lib 1.10
+     -
```

Comment:
This appears to have been missed when "MongoDB 5.1" was added to the table in https://github.com/mongodb/docs-ecosystem/commit/a88ae6a18a8c8c36bb566c01cba26a4a4d054441#diff-737154321a39ba198ec04d4a10f0e5ac581e1846ca91741294945b6c581a0e5dR7

cc: @ccho-mongodb